src/emmy/tape.cljc

```diff
@@ -455,6 +495,32 @@
 ;; values like maps or quaternions. [[emmy.differential/extract-tangent]] does
 ;; this for forward-mode, I believe.
 
+(declare ->partials)
+
+(defn- ->partials-fn
```

Comment:
This is the solution to Alexey's Amazing Bug, along with the use of `tag+perturbation` in `lift-2` below. Notice how similar it is to `extract-tangent-fn`... these ideas should be merged, and IDEALLY the tapes could use `extract-tangent` too. Still thinking about this.